net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Client.xml

```diff
@@ -12,8 +12,8 @@
                 <name type="TextField">
```

Comment:
maybe the `HostnameField` type is a better alternative here

Reply:
Good call out. I will do some testing with HostnameField

Reply:
```HostnameField``` validation seems to be working well.